gulpfile.js

```diff
@@ -528,11 +526,33 @@ gulp.task(
       )
       .on("error", function(e) {
         console.log(e);
+        throw e;
       })
       .on("end", cb);
   }
 );
 
+gulp.task(
+  "protractor-go:ci",
+  ["clean:test-reports", "clean:test-screenshots", "webdriver_update"],
+  function(cb) {
+    setTimeout(function () {
+      gulp
+        .src([])
+        .pipe(
+          protractor({
+            configFile: "./app/tests/e2e/e2e-conf.js"
+          })
+        )
+        .on("error", function(e) {
+          console.log(e);
+          throw e;
+        })
+        .on("end", cb);
```

Comment:
Is this giving 3 minutes to each test in order to perform ?
I modified tests that way a few days ago because the BE was exceptionally slow on my crappy laptop, but it is not meant to stay like that. BE is supposed to become faster, on the server that runs the tests for CI should be also much faster than my laptop.
3 minutes would not be an acceptable performance for any users, so I think we should lower that. In practice, what was the perf that you observed ?

Reply:
@olivvv In CI tests I must run gulp browsersinc in background mode. It takes 2,5 mins to execute this script. To not to have errors I added extra 0,5 mins. So, in total we get 3 mins. 
We can say that 3 mins is a delay before the script with tests will be executed. 
Running the tests locally will work without this delay.

Reply:
oh ok, so it is not 3 min for each test but 3 min delay once for the whole test suite ?

Reply:
@olivvv Yep, exactly.